src/test/java/uk/gov/justice/digital/delius/integration/api/SwaggerValidator.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.delius.integration.api;
+
+import io.swagger.v3.parser.converter.SwaggerConverter;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("dev-seed")
```

Comment:
you can take this out I think, else it will load all the H2 data that takes about 10 seconds

Reply:
:+1: will do in next PR